core/server/web/parent-app.js

```diff
@@ -38,10 +38,9 @@ module.exports = function setupParentApp(options = {}) {
     // Mount the  apps on the parentApp
     // API
     // @TODO: finish refactoring the API app
-    // @TODO: decide what to do with these paths - config defaults? config overrides?
-    parentApp.use(urlUtils.getApiPath({version: 'deprecated'}), require('./api/v0.1/app')());
-    parentApp.use(urlUtils.getApiPath({version: 'active', type: 'content'}), require('./api/v2/content/app')());
-    parentApp.use(urlUtils.getApiPath({version: 'active', type: 'admin'}), require('./api/v2/admin/app')());
+    parentApp.use(urlUtils.getApiPath({version: 'v0.1'}), require('./api/v0.1/app')());
+    parentApp.use(urlUtils.getApiPath({version: 'v2', type: 'content'}), require('./api/v2/content/app')());
+    parentApp.use(urlUtils.getApiPath({version: 'v2', type: 'admin'}), require('./api/v2/admin/app')());
```

Comment:
What about the other usages in the code base? e.g. in API v0.1
Just asking, because you updated this usage.

Reply:
So only place its used in v0.1 I think is the location header, which calls `deprecated`, that def needs to be updated 👍 The other 2 places we `deprecated` directly is `serve-public-files` and initializing scheduler. They are not directly v0.1 though, so not sure if its a good idea to update them 🤔

Reply:
Sounds good, i was mostly interested in the location header usage 👍